packages/teams-js/test/private/remoteCamera.spec.ts

```diff
@@ -61,6 +62,23 @@ describe('remoteCamera', () => {
 
     Object.values(FrameContexts).forEach(context => {
       if (allowedContexts.some(allowedContexts => allowedContexts === context)) {
+        it(`remoteCamera.getCapableParticipants should throw error when remoteCamera is not supported. context : ${context}`, async () => {
```

Comment:
I think there should be at least one test  case where isSupported returns true and we set the runtimeConfig explicitly. Can you please add one if it's not present?